bidfx/pricing/_pixie/pixie_provider.py

```diff
@@ -107,7 +108,7 @@ def _session_connection_attempt(self):
             self._publish_provider_status(ProviderStatus.READY)
             self._price_server_read_loop()
         except Exception as e:
-            log.warning(f"connection attempt failed due to: {e}")
+            log.warning(f"connection attempt failed due to: {traceback.format_exc()}")
```

Comment:
I'd like to understand what this traceback call provides. I hope it is not a stack trace. That would be inappropriate for a message intended for client consumption.

Reply:
it was for a stack trace, I thought the initial error message was too brief -- will reverse it